config-cli/src/main/java/com/quorum/tessera/config/cli/OverrideUtil.java

```diff
@@ -28,10 +29,12 @@
 
 public interface OverrideUtil {
```

Comment:
Can we have a header explaining what this class is for?
I shouldn't have to go through the code to work out the purpose of it.